oak_functions/sdk/oak_functions/src/lib.rs

```diff
@@ -183,3 +92,21 @@ pub fn result_from_status<T>(status: i32, val: T) -> Result<T, OakStatus> {
         None => Err(OakStatus::Unspecified),
     }
 }
+
+#[no_mangle]
+pub extern "C" fn alloc(len: u32) -> *mut u8 {
```

Comment:
I guess this means that the examples don't have to implement their own `alloc` function. If so, maybe explain in the docs that we have an implementation of `alloc` in our Rust SDK? Or add a doc comment here?

Reply:
Done in https://github.com/project-oak/oak/pull/2010